app/routes/docs/newproducers.mdx

```diff
@@ -0,0 +1,75 @@
+---
+handle:
+  breadcrumb: New Producers
+---
+
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+  SummaryBox,
+} from '@trussworks/react-uswds'
+import { GitHubLabel } from '~/components/GitHubLabel'
+
+# Setting up New GCN Notice Producers
+
+The following steps guide new instrument, mission, or observatory producers
+into setting up new notices streams that are distributed to the user community
+via [Kafka](faq#what-is-kafka). This process requires interactions
+with the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+to enable these accounts and topics are setup on the GCN Kafka broker,
+and we are happy to work together to help you define schema utilizing our
+core JSON schema.
+
+## Accounts and Scope
+
+Make sure that all of the producer users (who will enable GCN notices in your pipelines)
+have signed in (at least once) to
+https://test.gcn.nasa.gov and https://gcn.nasa.gov.
+
+Please send the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+a list of email addresses of all the people who should have the
+producer access to the MissionName.\* topics.
+
+## Defining Kafka Topics and JSON Schema
+
+The preferred format for formatting of the data sent via GCN Kafka is JSON.
+We have built a set of core schema which serves as the building blocks for
+your Notices. You can also define instrument-specific quantities in your
+schema, but we request that you utilize the core schema as much as possible.
+Please add your schema to the [GCN repository](https://github.com/nasa-gcn/gcn-schema).
+
+The GCN team is happy to iterate with the producers on their schema contents and format.
+Your pipeline will generate JSON files following these schema and send to GCN
+in the following process.
+
+## Steps to Start Publishing Alerts
+
+1. Log out and log back in.
+2. Go through the "Start Streaming GCN Notices" process.
+3. On Step 2 of "Start Streaming GCN Notices", choose the scope gcn.nasa.gov/kafka-MissionName-producer.
+4. Your producer code will look very similar to the client example code.
+   Start from this and adjust as you see fit:
+
+```sh
+from gcn_kafka import Producer
+# Connect as a producer. <br>
+# Warning: don't share the client secret with others.
+producer = Producer(client_id='fill me in', _secret='fill me in')
+topic = 'MissionName.example'
+# anything starting with 'MissionName'
+data = b'...'  # any bytes
+producer.produce(topic, data)
+```
+
+## Mission page
+
+Add mission page linking to relevant information and archives, and description of
+new GCN notices content. GCN Team is also available to help construct and format
```

Comment:
"You can create a new mission page by submitting a pull request (add link) or by emailing text to the GCN Team (insert link)."